index.html

```diff
@@ -31,105 +35,124 @@
-                      <input id="Option5" type="checkbox" autocomplete="off"><label class="chk" for="Option5">1/2 cup brown sugar</label>
-                      <input id="Option6" type="checkbox" autocomplete="off"><label class="chk" for="Option6">2-4 tbsp water</label>
-                    </form>
-                    <form class="listright">
-                      <input id="Option7" type="checkbox" autocomplete="off"><label class="chk" for="Option7">1 1/2 cups heavy cream</label>
-                      <input id="Option8" type="checkbox" autocomplete="off"><label class="chk" for="Option8">1/3 cup granulated sugar</label>
-                      <input id="Option9" type="checkbox" autocomplete="off"><label class="chk" for="Option9">2 egg yolks</label>
-                      <input id="Option10" type="checkbox" autocomplete="off"><label class="chk" for="Option10">1 lemon, juiced</label>
-                      <input id="Option11" type="checkbox" autocomplete="off"><label class="chk" for="Option11">2 tbsp butter</label>
-                      <input id="Option12" type="checkbox" autocomplete="off"><label class="chk" for="Option12">1 cup honey roasted pecans, roughly chopped</label>
-                    </form>
-                  </section>
-                </div>    <!--END twelve.col-->
-              </div>      <!--END row-->
-            </div>        <!--END container-->
-
-            <div class="container">
-              <div class="row">
-                <div class="twelve columns">
-                  <aside class="author_notes">
-                    <img src="imgs/headshot.png" alt="vanessa headshot" class="left headshot">
-                    <h3>Vanessa Stevenson</h3>
-                    <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
-                    <a href="#" class="share">SHARE RECIPE</a>
-                  </aside>
-                </div>    <!--END twelve.col-->
-              </div>      <!--END row-->
-            </div>        <!--END container-->
+        <main class="mainbody container">
+          <article class="mainarticle container">
+            <div class="row">
+              <div class="six columns">
+                <h1>Chocolate Pizza</h1>
+                <h2>POSTED ON 15 DEC 2013 / DESSERTS</h2>
+              </div>    <!--END six.col-->
+              <div class="six columns">
+                <p class="right print" id="print"><span class="icon-printer"></span> PRINT</p>
+              </div>    <!--END six.col-->
+            </div>      <!--END row-->
+            <div class="row">
+              <div class="twelve columns">
+                <img src="imgs/chocopizza.jpg" alt="chocolate pizza" class="mainpic">
+              </div>    <!--END twelve.col-->
+            </div>      <!--END row-->
+            <div class="row">
+              <div class="twelve columns">
+                <section class="bodyp">
```

Comment:
This section.bodyp and div.twelve.columns could be combined into a single wrapper